draft-ietf-rats-architecture.md

```diff
@@ -877,7 +878,41 @@ are signed.  For example, values might have been generated at boot, and then
 used in claims as long as the signer can guarantee that they cannot have changed
 since boot.
 
-A more detailed discussion with examples appears in {{time-considerations}}.
+The term freshness is applied to both Evidence and Attestation Results in
+this document. Unfortunately, the term freshness is also a bit overloaded and
+in order to illustrate the application of the term in this document better,
+two characteristics that can be part of freshness are elaborated on below:
+recentness and uniqueness:
+
+Recentness:
+
+: a quality of Evidence that is based on the time that has passed between
+creation of the evidence by an Attester (typically by "sealing" a Claims set
+via signing) and the reception of that Evidence by a Verifier or (relaying
+party).
+
+: Evidence can be not recent enough, if the Evidence created from collected
+Claims is too old. A nonce supplied during a Challenge/Response handshake and
+a time-stamp based proof-of-recentness between time-synchronized peers are
+both usage scenarios in which recentness characteristics are provided to the
+Verifier.
+
+Uniqueness:
```

Comment:
I am not sure yet why this section is important.  I read the text below and I still can't figure out why it would be in the document.  I can't say I object yet, I can just say "why do we need this"?